### PR TITLE
[GSW-410] fix: Swap Improve UI

### PR DESCRIPTION
**Pull request by khiemldk**

### CODE REVIEW

**Review by jinoosss — COMMENTED** [review]
I left a few comments.

File: packages/web/src/components/common/wallet-connector-button/WalletConnectorButton.tsx [hunk 1]
```diff
@@ -49,7 +81,13 @@ const WalletConnectorButton: React.FC<WalletConnectProps> = ({
         <Button
           leftIcon={
             account && account.chainId !== CHAIN_ID ? (
```
[the remark this answers is not in the document]
Comment (khiemldk) [reply packages/web/src/components/common/wallet-connector-button/WalletConnectorButton.tsx]: Updated in use-wallet hook

[the remark this answers is not in the document]
Comment (khiemldk) [reply packages/web/src/components/common/wallet-connector-button/WalletConnectorButton.tsx]: Updated get isSwitchNetwork in use-wallet for all

**Review by jinoosss — APPROVED** [review]
👍

**Pull request merged**